OracleTuxedo/core/dockerfiles/12.1.3/Dockerfile

```diff
@@ -7,25 +7,25 @@
 #
 
 # Pull base image
-FROM oraclelinux
+FROM container-registry.oracle.com/java/serverjre
 
 MAINTAINER Todd Little <todd.little@oracle.com>
 
 # Core install doesn't include unzip or gcc, add them
 # Create the installation directory tree and user oracle with a password of oracle
-RUN yum -y install unzip gcc file; yum -y clean all && \
+RUN yum -y install unzip gcc file which; yum -y clean all && \
     groupadd -g 1000 oracle; useradd -b /home -m -g oracle -u 1000 -s /bin/bash oracle && \
     echo oracle:oracle | chpasswd; echo root:samplesvm | chpasswd
 
-COPY install.sh tuxedo12.1.3.rsp p*_121300_Linux-x86-64.zip tuxedo121300_64_Linux_01_x86.zip /home/oracle/Downloads/
+COPY install.sh tuxedo12.1.3.rsp  tuxedo121300_64_Linux_01_x86.zip /home/oracle/Downloads/
 #ADD simpapp_runme.sh start_tlisten.sh /home/oracle/
 
 RUN chown oracle:oracle -R /home/oracle
 WORKDIR /home/oracle/Downloads
```

Comment:
You shouldn't be working in the home directory. I would recommend using either `/u01/` or `/opt`